includes/options/class-amp-options-manager.php

```diff
@@ -562,6 +567,20 @@ public static function persistent_object_caching_notice() {
 		}
 	}
 
+	/**
+	 * Render the Stories deprecation admin notice.
+	 */
+	public static function render_stories_deprecation_notice() {
+		if ( 'toplevel_page_' . self::OPTION_NAME !== get_current_screen()->id ) {
+			return;
+		}
+
+		printf(
+			'<div class="notice notice-warning"><p>%s</p></div>',
+			esc_html__( 'The Stories experience in the AMP plugin has been deprecated and will no longer be supported. A separate plugin for Stories be available soon for testing. Please backup or export your existing Stories as they will not be available in the next version of the AMP plugin.', 'amp' )
+		);
```

Comment:
If the Stories experience is hidden, then instead of a warning there should be info notice to say something like:

> The Stories experience has been removed from the AMP plugin. This beta feature is being split into a separate standalone plugin which will be available for installation soon.

Reply:
Done in https://github.com/ampproject/amp-wp/pull/4219/commits/2dd83a06d081a03f32bfd2532b9706d53a52426e.